rsrcdump/structtemplate.py

```diff
@@ -113,7 +117,7 @@ def tag_values(self, values: tuple[str | int, ...]) -> dict[str, str | int] | st
             # Multiple-element structure but no field names: return the tuple
             return values
 
-    def pack(self, obj: list[str] | str) -> bytes:
+    def pack(self, obj: list[JSONScalar] | dict[str, JSONScalar] | JSONScalar) -> bytes:
```

Comment:
How about defining a shorthand for `list[JSONScalar] | dict[str, JSONScalar] | JSONScalar`, since we're repeating it a few times in the file? Maybe we can call it JSONPackable or something. I'm open to other names if you have better ideas

Reply:
Added type alias in 0e63227